Cython/Build/Tests/TestCyCache.py

```diff
@@ -7,6 +7,7 @@
 import unittest
```

Comment:
Not sure whether this TestCyCache module should not be placed somewhere else since it is not related to Build package only anymore...

Reply:
Or cache module will be moved to `Build` directory - see: https://github.com/cython/cython/pull/6090#discussion_r1527455984

Reply:
Now when `Cache.py` is in `Build`directory this comment is not valid anymore I think...